Amazon.QLDB.Driver/session/QldbSession.cs

```diff
@@ -188,7 +193,6 @@ public virtual ITransaction StartTransaction()
             }
             catch (BadRequestException e)
             {
-                this.NoThrowAbort(null);
```

Comment:
Can we add a unit test for this?

Reply:
There has been a test case. Line 263 of QldbSessionTest.cs.